guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -15,46 +15,65 @@ New hosts are provisioned with a boot disk or PXE-less discovery image that {Pro
 
-
 [[Provisioning_Bare_Metal_Hosts-Prerequisites_for_Bare_Metal_Provisioning]]
 === Prerequisites for Bare Metal Provisioning
 
 The requirements for bare metal provisioning include:
 
-  * Synchronized content repositories for the version of Red Hat Enterprise Linux that you want to use. For more information, see link:/html/content_management_guide/importing_red_hat_content#Importing_Red_Hat_Content-Synchronizing_Red_Hat_Repositories[Synchronizing Red Hat Repositories] in the _Content Management Guide_.
+ifeval::["{Build}" == "foreman"]
+  * The installation media that you require for the operating systems you want to use to provision using Amazon EC2.
+endif::[]
+ifeval::["{Build}" == "satellite"]
+  * Synchronized content repositories for Red{nbsp}Hat Enterprise Linux. For more information, see link:/html/content_management_guide/importing_red_hat_content#Importing_Red_Hat_Content-Synchronizing_Red_Hat_Repositories[Synchronizing Red{nbsp}Hat Repositories] in the _Content Management Guide_.
+endif::[]
   * A {SmartProxyServer} managing the network for bare metal hosts. For unattended provisioning and discovery-based provisioning, {ProjectServer} requires PXE server settings. For more information, see xref:Configuring_Networking[].
+ifeval::["{Build}" == "satellite"]
   * An activation key for host registration. For more information, see link:/html/content_management_guide/managing_activation_keys#Managing_Activation_Keys-Creating_an_Activation_Key[Creating An Activation Key] in the _Content Management_ guide.
-  * A blank bare metal host.
+endif::[]
+ifeval::["{Build}" == "foreman"]
+  * If the Katello plugin is installed, an activation key for host registration. For more information, see link:/html/content_management_guide/managing_activation_keys#Managing_Activation_Keys-Creating_an_Activation_Key[Creating An Activation Key] in the _Content Management_ guide.
+endif::[]
+  * A bare metal host or a blank VM.
+
+NOTE: PXE workflows can be used to provision virtual machines which are not under {Project} control as well.
 
 For information about the security token for unattended and PXE-less provisioning, see xref:Provisioning_Bare_Metal_Hosts-Security_Token_in_the_Boot_Process[].
 
 [[Provisioning_Bare_Metal_Hosts-Security_Token_in_the_Boot_Process]]
 === Configuring the Security Token Validity Duration
 
-When performing unattended and PXE-less provisioning, as a security measure, {Project} automatically generates a unique token and adds this token to the URL of the ISO image that downloads during the Kickstart provisioning process.
+When performing unattended and PXE-less provisioning, as a security measure, {Project} automatically generates a unique token and adds this token to the installation recipe URL which appears in PXE configuration file (PXELinux, Grub2).
+ifeval::["{build}" == "satellite"]
+For {RHELServer} installation recipe is also known as kickstart.
+endif::[]
```

Comment:
This is tough, we very often use kickstart term, however upstream kickstart is actually an "installation recipe". Other types are "preseed" etc.

Reply:
`:provision-script: kickstart` it is.